docs/schema-loader-import.md

```diff
@@ -0,0 +1,272 @@
+# Importing existing tables to ScalarDB using ScalarDB Schema Loader
+
+You might want to use ScalarDB (e.g., for database-spanning transactions) with your existing databases. In that case, you can import those databases under the ScalarDB control using ScalarDB Schema Loader. ScalarDB Schema Loader automatically adds ScalarDB-internal metadata columns in each existing table and metadata tables to enable various ScalarDB functionalities including transaction management across multiple databases.
+
+## Before you begin
+
+{% capture notice--warning %}
+**Attention**
+
+You should carefully plan to import a table to ScalarDB in production because it will add transaction metadata columns to your database tables and the ScalarDB metadata tables. There would also be several differences between your database and ScalarDB and limitations.
```

Comment:
@josh-wong Do you mean the following change?

```suggestion
You should carefully plan to import a table to ScalarDB in production because it will add transaction metadata columns to your database tables and the ScalarDB metadata tables. In this case, there would also be several differences between your database and ScalarDB, as well as some limitations.
```

Reply:
Oops! Yes, that is correct. Copy-paste error...😞

Reply:
No problem! Thank you!